front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -249,50 +249,19 @@ const AdminProjectEventEdit = () => {
     setAttendanceLimitVisible(toggleValue);
     setAttributeDiff({
       ...attributeDiff,
-      maximum_attendees: null,
+      maximum_attendees: toggleValue === false ? null : 100,
```

Comment:
If we turn on the toggle, it makes sense to set a default value. 

(I'm not sure if 100 is a sensible default. Maybe a lower number is better? 20? 50?)